### PR TITLE
Documentation : Improve doc for the CLI (2) #551

### PR DESCRIPTION
**Pull request by cserf**
Documentation : Improve doc for the CLI (2) #551

### CODE REVIEW

File: doc/source/cli_admin_examples.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
+===============
+Rucio admin CLI
```
Comment (mlassnig) [inline doc/source/cli_admin_examples.rst]: Rucio Administration CLI

File: doc/source/cli_admin_examples.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+===============
+
+
+Rucio provides an admin CLI. The get methods can be executed by any users, but the set methods require some admin privileges::
```
Comment (mlassnig) [inline doc/source/cli_admin_examples.rst]: Rucio provides a CLI for administrative tasks.

File: doc/source/cli_admin_examples.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                          CA certificate to verify peer against (SSL)
+
+Account and identity methods
+============================
+
+To create a new account::
+
+  $ rucio-admin account add --type USER --email jdoe@blahblih.com jdoe
+
+You can choose different types in the list USER, GROUP, SERVICE. Once the account is created, you need to create and attach an identity to this account::
```
Comment (mlassnig) [inline doc/source/cli_admin_examples.rst]: You can choose different account types (USER, GROUP, SERVICE). USER is ...., GROUP is ..., SERVICE is ....

File: doc/source/cli_admin_examples.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+  $ rucio-admin identity add --type X509 --id "/DC=blah/DC=blih/OU=Organic Units/OU=Users/CN=jdoe" --email jdoe@blahblih.com --account jdoe
+
+The list of possible identity is X509, GSS, USERPASS, SSH::
```
Comment (mlassnig) [inline doc/source/cli_admin_examples.rst]: .. identity types ...

File: doc/source/cli_admin_examples.rst [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+And list these attributes::
+
+  $ rucio-admin account list-attributes jdoe
+  +---------+-------+
+  | Key     | Value |
+  |---------+-------|
+  | country | xyz   |
+  +---------+-------+
+
+You can also list all the account matching a certain attribute using the filter option::
```
Comment (mlassnig) [inline doc/source/cli_admin_examples.rst]: accounts

File: doc/source/cli_admin_examples.rst (hunk 3 above)
Comment (cserf) [reply doc/source/cli_admin_examples.rst]: is... ?

File: doc/source/cli_admin_examples.rst [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+                          Set all timeout values to SECONDS
+    -u USERNAME, --user USERNAME
+                          username
+    -pwd PASSWORD, --password PASSWORD
+                          password
+    --certificate CERTIFICATE
+                          Client certificate file
+    --ca-certificate CA_CERTIFICATE
+                          CA certificate to verify peer against (SSL)
+
```
Comment (vingar) [inline doc/source/cli_admin_examples.rst:48]: this part can be generated in a distinct section (see #627)

Comment (cserf) [reply doc/source/cli_admin_examples.rst:48]: OK. Maybe in the meantime we approve this PR, then I will resubmit one with improved doc in the bin

Comment (bari12) [reply doc/source/cli_admin_examples.rst:48]: 👍

**Review by bari12 — APPROVED** [review]

**Pull request merged**